.github/comment.yml

```diff
@@ -122,8 +122,15 @@
     variables:
       openapiHub: https://aka.ms/openapihub
     keywords:
-      - "I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow."
```

Comment:
I am kind of surprised this comment continued being added in newer PRs, [because I have since removed this sentence](https://github.com/Azure/azure-rest-api-specs/blob/0799c57684fa0f9d55230554dc51dc00aad99a0b/.github/PULL_REQUEST_TEMPLATE/control_plane_template.md?plain=1#L17C87-L17C87) replacing [it with the new one](https://github.com/Azure/azure-rest-api-specs/blob/89e91a02e7cf20da48bb84c34eda694faed4dc5c/.github/PULL_REQUEST_TEMPLATE/control_plane_template.md?plain=1#L18C7-L18C103). Here is an example of this comment being added with the new sentence in place.

Reply:
Were you able to track down why this was happening still? Do we need to restart the bot to get it to detect new changes or maybe there were some older nodes still using the older configuration?

Reply:
I didn't try to track it down yet, but I'll look into it at some point. Other changes I made to the `comment.yml` were picked up by the bot, like different requirements to add given comment and different comment contents.

Reply:
@weshaggard OK I think I know what's up. The `keywords` property is ignored because the [rule type is `PROpen`](https://github.com/Azure/azure-rest-api-specs/blob/9349c36be83441a1092c0a93bd501be39a8ad190/.github/comment.yml#L121C17-L121C17) and not `checkbox`.

The code reading the keywords is in [workflowBot.ts / handlePullRequestEdited / getMatchedRuleByCheckboxText](https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_git/openapi-alps?path=private/openapi-kebab/src/bots/workflow/workflowBot.ts&version=GC9df1258de7cdcbd322cfc2ff15f4ef4f96998f9d&line=857&lineStartColumn=11&lineEndColumn=39&_a=contents). It requires for the rule type to be `checkbox`, [as seen here](https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_git/openapi-alps?path=private/openapi-kebab/src/bots/workflow/workflowBot.ts&version=GC9df1258de7cdcbd322cfc2ff15f4ef4f96998f9d&line=858&lineStartColumn=34&lineEndColumn=85&_a=contents). 

The `PROpen` rule is handled by [`handlePullRequestOpened`](https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_git/openapi-alps?path=private/openapi-kebab/src/bots/workflow/workflowBot.ts&version=GC9df1258de7cdcbd322cfc2ff15f4ef4f96998f9d&line=448&lineStartColumn=16&lineEndColumn=39&_a=contents) function, [which ends up calling `handlePullRequestEdited`](https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_git/openapi-alps?path=private/openapi-kebab/src/bots/workflow/workflowBot.ts&version=GC9df1258de7cdcbd322cfc2ff15f4ef4f96998f9d&line=581&lineStartColumn=5&lineEndColumn=49&_a=contents), but doesn't process the `keywords` if the type is not `checkbox`.

I will remove this invalid `keywords` line.